orion/sites/admin/src/announce/announce.js

```diff
@@ -2,12 +2,70 @@
 require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
+import { Link } from 'react-router-dom';
 
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 3; 	/* this needs to be a variable later */
+
+		/* these fake announcements will eventually be replaced with real announcements */
+		let fakeAnnounceA = {
+			date: "02/15/2020",
+			author: "Melon King",
+			message: "Only evil people waste watermelon"
+		};
+		let fakeAnnounceB = {
+			date: "02/16/2020",
+			author: "Fruit Hater",
+			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
+		};
+		/* will use this to test text wrapping later */
+		let fakeAnnounceC = {
+			date: "02/23/2020",
+			author: "Talkative",
+			message: "This will be a longer message later"
+		};
 		return (
       <div id="view-announce">
-        <h1>All Announcements</h1>
+        <h1>All Announcements ({numAnnouncements}) </h1>
+				<ul> {/* this is the subheading */}
```

Comment:
I think instead of leaving comments, we can try using "meaningful code" to express what we're trying to say in comments.
For example, this `<ul>` is a subheader.
So we can give it an id like so:
```
<ul id="subheader">
```
That way we don't need verbose comments :)